cli/src/pcluster/commands.py

```diff
@@ -544,3 +544,16 @@ def _get_default_template_url(region):
             REGION=region, SUFFIX=".cn" if region.startswith("cn") else "", VERSION=utils.get_installed_version()
         )
     )
+
+
+def build_image(args):
+    """Build AWS ParallelCluster AMI."""
+    LOGGER.info("Building AWS ParallelCluster AMI. This could take a while...")
+    response = PclusterApi().build_image(
+        imagebuilder_config=load_yaml_dict(args.config_file),
```

Comment:
let's add a try catch because the load_yaml_dict can fail.
In the clsuter create we are catching it in the _parse_config_file function.